packages/adapters/ethers/src/client.ts

```diff
@@ -828,8 +822,8 @@ export class EthersAdapter {
     }
 
     this.appKit?.setLoading(true)
-    const chainId = NetworkUtil.caipNetworkIdToNumber(this.appKit?.getCaipNetwork()?.id)
-    this.appKit?.setCaipAddress(`eip155:${chainId}:${address}`, this.chainNamespace)
+    const chainId = this.appKit?.getCaipNetwork()?.id
+    this.appKit?.setCaipAddress(`${this.chainNamespace}:${chainId}:${address}`, this.chainNamespace)
```

Comment:
is this change ok? won't it break anything?

Reply:
Nope, we should avoid hard coded strings